src/litserve/__main__.py

```diff
@@ -11,16 +11,50 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from jsonargparse import set_config_read_mode, set_docstring_parse_options, CLI
-
+from argparse import ArgumentParser
 from litserve.docker_builder import dockerize
 
 
 def main():
-    cli_components = {"dockerize": dockerize}
-    set_docstring_parse_options(attribute_docstrings=True)
-    set_config_read_mode(urls_enabled=True)
-    CLI(cli_components)
+    parser = ArgumentParser(description="CLI for LitServe")
+    subparsers = parser.add_subparsers(dest="command", help="Sub-command help")
+
+    # dockerize sub-command
+    dockerize_parser = subparsers.add_parser(
+        "dockerize",
+        help=dockerize.__doc__,
+        description="Generate a Dockerfile for the given server code.",
+    )
+    dockerize_parser.add_argument(
+        "server_filename",
+        type=str,
+        help="The path to the server file. Example: server.py or app.py.",
+    )
+    dockerize_parser.add_argument(
+        "--port",
+        type=int,
+        default=8000,
+        help="The port to expose in the Docker container. Defaults to 8000.",
+    )
+    dockerize_parser.add_argument(
+        "--gpu",
+        default=False,
+        action="store_true",
+        help="Whether to use a GPU-enabled Docker image. Defaults to false.",
+    )
+    dockerize_parser.set_defaults(func=lambda args: dockerize(args.server_filename, args.port, args.gpu))
+
+    # Add "litserve build or litserve serve" sub-commands here
```

Comment:
remove comments?

Reply:
ah, I thought it would be helpful for future contributors. Will drop it.